install_config/master_node_configuration.adoc

```diff
@@ -1743,10 +1743,12 @@ To launch a node server using a node configuration file:
 $ openshift start node --config=/openshift.local.config/node-<node_hostname>/node-config.yaml
 ----
 
-[[master-node-config-logging-levels]]
-== Configuring Logging Levels
+[[master-node-view-logs]]
+== Viewing Master and Node Logs
+
+{product-title} uses the `systemd-journald.service` to collect log messages for debugging the masters and nodes.
```

Comment:
> `systemd-journald.service\` ... for ... **masters and** nodes.

This is still not accurate

Reply:
@xingxingxia Is "{product-title} uses the `systemd-journald.service` to collect log messages for debugging." This is in the current docs. Or, is your concern with `systemd-journald.service`